frontend/locales/en.json

```diff
@@ -66,6 +66,7 @@
       "nominators": "Nominators",
       "accounts": "Accounts",
       "faucet": "Faucet",
+      "cere-chart": "Chart",
```

Comment:
Why do we need `cere` prefix here? We are going to have few charts. Rename to plural.

Reply:
updated